numba_cuda/numba/cuda/codegen.py

```diff
@@ -271,7 +274,7 @@ def get_cubin(self, cc=None):
             return cubin
 
         if self._lto and config.DUMP_ASSEMBLY:
-            linker = driver.Linker.new(
+            linker = driver._Linker.new(
```

Comment:
Note / nit: this feels a little funny because the `new` method is a class method of `_LinkerBase` rather than `_Linker`.